code/modules/reagents/Chemistry-Goon-Pyrotechnics.dm

```diff
@@ -141,4 +131,44 @@ proc/goonchem_vortex(var/turf/simulated/T, var/setting_type, var/range, var/pull
 						step_away(X,T)
 				else
 					for(var/i = 0, i < pull_times, i++)
-						step_towards(X,T)
\ No newline at end of file
+						step_towards(X,T)
+
+/datum/reagent/blackpowder
+	name = "Black Powder"
+	id = "blackpowder"
+	description = "Explodes. Violently."
+	reagent_state = LIQUID
+	color = "#000000"  //rgb: 96, 165, 132
+	metabolization_rate = 0.05
+
```

Comment:
if you want to go all out, I recommend making a new cleanable decal that creates a very small explosion when heated to the proper temperature or disturbed by a vortex.

By the by, body temperature probably should impact a chem's temp; the whole reason black powder has a low metabolism rate is so you can inject it into someone subtly, then ignite them without them realizing what's going on until it's too late.